### PR TITLE
Translate topics/registries.md into Korean

### PR DESCRIPTION
**Pull request by markruler**
Translate topics/registries.md into Korean

- https://helm.sh/docs/topics/registries/

Signed-off-by: markruler <csu0414@gmail.com>

### CODE REVIEW

**Comment by markruler** [comment]
```console
ERROR 2020/09/15 16:37:18 [ko] REF_NOT_FOUND: Ref "chart_repository.md": "/opt/build/repo/content/ko/docs/topics/registries.md:236:30": page not found
```

If #824 is merged, this error will be eliminated.

File: content/ko/docs/topics/registries.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+헬름 3는 패키지 배포를 위해 <a href="https://www.opencontainers.org/"
+target="_blank">OCI</a>를 지원한다. 차트 패키지는 OCI 기반 레지스트리 간에
+저장 및 공유할 수 있다.
+
+## OCI 지원 활성화
+
+현재 OCI 지원은 *실험적(experimental)* 으로 간주되고 있다.
+
+아래에서 서술할 명령들을 사용하려면 환경에서 `HOME_VERSION_OCI` 변수를 설정하자:
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:16]:
```suggestion
아래에서 기술된 명령들을 사용하려면 환경에서 `HOME_VERSION_OCI` 변수를 설정하자:
```

File: content/ko/docs/topics/registries.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+```console
+export HELM_EXPERIMENTAL_OCI=1
+```
+
+## 레지스트리 실행
+
+테스트 목적으로 레지스트리를 시작하는 것은 쉽다. 도커를 설치한 경우 다음 명령을
+실행하십시오.
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:25]:
```suggestion
테스트 목적으로 레지스트리를 시작하는 것은 간단하다. 도커를 설치한 경우, 다음 명령어를
실행하자.
```

명령문은 청유형으로 표현하기로 한 관례가 있는 걸로 알고 있습니다.

File: content/ko/docs/topics/registries.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+
+```console
+docker run -dp 5000:5000 --restart=always --name registry registry
+```
+
+이렇게 하면 `localhost:5000`에서 레지스트리 서버가 시작된다.
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:31]:
```suggestion
이렇게 하면 `localhost:5000`에서 레지스트리 서버가 시작될 것이다.
```

File: content/ko/docs/topics/registries.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+
+`docker logs -f registry`를 사용하여 로그를 확인하고 `docker rm -f registry`를
+사용하여 중지하자.
+
+스토리지를 유지하려면 위의 명령에 `-v $(pwd)/registry:/var/lib/registry`를
+추가하자.
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:37]:
```suggestion
스토리지를 보존하려면 위의 명령에 `-v $(pwd)/registry:/var/lib/registry`를
추가하자.
```

File: content/ko/docs/topics/registries.md [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+
+자세한 설정 옵션은 [해당 문서](https://docs.docker.com/registry/deploying/)를
+참조하자.
+
+### 인증
+
+레지스트리에서 인증을 실행하려면 다음을 수행한다-
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:44]:
```suggestion
레지스트리에 인증을 활성화하려면, 다음과 같이 해보자.
```

File: content/ko/docs/topics/registries.md [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+
+먼저 사용자 이름과 비밀번호로 `auth.htpasswd` 파일을 만들자.
+
+```console
+htpasswd -cB -b auth.htpasswd myuser mypass
+```
+
+그런 다음 서버를 시작하여 해당 파일을 마운트하고 `REGISTRY_AUTH` 환경 변수를
+지정하자:
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:53]:
```suggestion
그런 다음, 서버를 시작하여 해당 파일을 마운트하고 `REGISTRY_AUTH` 환경 변수를
지정하자.
```

File: content/ko/docs/topics/registries.md [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+
+```console
+docker run -dp 5000:5000 --restart=always --name registry \
+  -v $(pwd)/auth.htpasswd:/etc/docker/registry/auth.htpasswd \
+  -e REGISTRY_AUTH="{htpasswd: {realm: localhost, path: /etc/docker/registry/auth.htpasswd}}" \
+  registry
+```
+
+## 레지스트리 작업 명령
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:62]:
```suggestion
## 레지스트리 작업 명령어
```

File: content/ko/docs/topics/registries.md [hunk 8]
```diff
@@ -0,0 +1,237 @@
...
+
+`helm registry`와 `helm chart` 명령에서 레지스트리와 로컬 캐시에 대한 작업을 할 수 있게 한다.
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:64]:
```suggestion
`helm registry`와 `helm chart` 명령어로 레지스트리와 로컬 캐시에 대한 작업을 할 수 있다.
```

File: content/ko/docs/topics/registries.md [hunk 9]
```diff
@@ -0,0 +1,237 @@
...
+
+### `registry` 하위 명령
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:66]:
```suggestion
### `registry` 하위 명령어
```

File: content/ko/docs/topics/registries.md [hunk 10]
```diff
@@ -0,0 +1,237 @@
...
+#### `logout`
+
+레지스트리에서 로그아웃
+
+```console
+$ helm registry logout localhost:5000
+Logout succeeded
+```
+
+### `chart` 하위 명령
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:87]:
```suggestion
### `chart` 하위 명령어
```

File: content/ko/docs/topics/registries.md [hunk 11]
```diff
@@ -0,0 +1,237 @@
...
+
+#### `save`
+
+차트 디렉토리를 로컬 캐시에 저장
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:91]:
```suggestion
로컬 캐시에 차트 디렉토리를 저장
```

File: content/ko/docs/topics/registries.md [hunk 12]
```diff
@@ -0,0 +1,237 @@
...
+localhost:5000/stable/acs-engine-autoscaler:2.2.2       acs-engine-autoscaler   2.2.2   d8d6762 4.3 KiB         2 hours
+localhost:5000/stable/aerospike:0.2.1                   aerospike               0.2.1   4aff638 3.7 KiB         2 hours
+localhost:5000/stable/airflow:0.13.0                    airflow                 0.13.0  c46cc43 28.1 KiB        2 hours
+localhost:5000/stable/anchore-engine:0.10.0             anchore-engine          0.10.0  3f3dcd7 34.3 KiB        2 hours
+...
+```
+
+#### `export`
+
+차트를 디렉토리로 내보내기
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:120]:
```suggestion
디렉토리로 차트 내보내기
```

File: content/ko/docs/topics/registries.md [hunk 13]
```diff
@@ -0,0 +1,237 @@
...
+digest:  1b251d38cfe948dfc0a5745b7af5ca574ecb61e52aed10b19039db39af6e1617
+size:    2.4 KiB
+name:    mychart
+version: 0.1.0
+Exported chart to mychart/
+```
+
+#### `push`
+
+원격으로 차트 밀어내기
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:134]:
```suggestion
원격지로 차트 밀어주기(push)
```

File: content/ko/docs/topics/registries.md [hunk 14]
```diff
@@ -0,0 +1,237 @@
...
+digest:  1b251d38cfe948dfc0a5745b7af5ca574ecb61e52aed10b19039db39af6e1617
+size:    2.4 KiB
+name:    mychart
+version: 0.1.0
+2.7.0: pushed to remote (1 layer, 2.4 KiB total)
+```
+
+#### `remove`
+
+캐시에서 차트 제거
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:149]:
```suggestion
캐시에서 차트 제거하기
```

File: content/ko/docs/topics/registries.md [hunk 15]
```diff
@@ -0,0 +1,237 @@
...
+
+```console
+$ helm chart remove localhost:5000/myrepo/mychart:2.7.0
+2.7.0: removed
+```
+
+#### `pull`
+
+원격에서 차트 당기기
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:158]:
```suggestion
원격지에서 차트 가져오기(pull)
```

File: content/ko/docs/topics/registries.md [hunk 16]
```diff
@@ -0,0 +1,237 @@
...
+2.7.0: Pulling from localhost:5000/myrepo/mychart
+ref:     localhost:5000/myrepo/mychart:2.7.0
+digest:  1b251d38cfe948dfc0a5745b7af5ca574ecb61e52aed10b19039db39af6e1617
+size:    2.4 KiB
+name:    mychart
+version: 0.1.0
+Status: Downloaded newer chart for localhost:5000/myrepo/mychart:2.7.0
+```
+
+## 나의 차트는 어디 있는가?
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:171]:
```suggestion
## 내 차트는 어디 있나?
```

File: content/ko/docs/topics/registries.md [hunk 17]
```diff
@@ -0,0 +1,237 @@
...
+
+위의 명령들을 사용하여 저장된 차트는 파일 시스템에 캐시된다.
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:173]:
```suggestion
위의 명령어들을 사용하여 저장된 차트는 파일 시스템에 캐시될 것이다.
```

File: content/ko/docs/topics/registries.md [hunk 18]
```diff
@@ -0,0 +1,237 @@
...
+
+[OCI 이미지 레이아웃
+사양](https://github.com/opencontainers/image-spec/blob/master/image-layout.md)은 파일 시스템 레이아웃을 엄격하게 따른다. 예를 들어,
+
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:177]:
```suggestion
[OCI 이미지 레이아웃
사양](https://github.com/opencontainers/image-spec/blob/master/image-layout.md)은
파일시스템 레이아웃에 엄격하게 적용되는데, 예를 들면 다음과 같다.
```

File: content/ko/docs/topics/registries.md [hunk 19]
```diff
@@ -0,0 +1,237 @@
...
+    │   │       ├── 1b251d38cfe948dfc0a5745b7af5ca574ecb61e52aed10b19039db39af6e1617
+    │   │       ├── 31fb454efb3c69fafe53672598006790122269a1b3b458607dbe106aba7059ef
+    │   │       └── 8ec7c0f2f6860037c19b54c3cfbab48d9b4b21b485a93d87b64690fdb68c2111
+    │   ├── index.json
+    │   ├── ingest
+    │   └── oci-layout
+    └── config.json
+```
+
+모든 헬름 차트 매니페스트에 대한 참조(ref)를 포함하는 index.json의 예시:
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:194]:
```suggestion
모든 헬름 차트 매니페스트에 대한 참조(ref)를 포함하는 예시 index.json:
```

File: content/ko/docs/topics/registries.md [hunk 20]
```diff
@@ -0,0 +1,237 @@
...
+      "size": 2487
+    }
+  ]
+}
+```
+
+## 차트 저장소에서 마이그레이션
+
+클래식 [차트 저장소]({{< ref path="chart_repository.md" lang="ko" >}}) (index.yaml 기반 저장소)에서 마이그레이션하는 작업은 `helm fetch` (헬름 2 CLI), `helm
+chart save`, `helm chart push`만큼 간단하다.
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:237]:
```suggestion
## 차트 저장소에서 마이그레이션

클래식 [차트 저장소]({{< ref path="chart_repository.md" lang="ko" >}}) (index.yaml 기반 저장소)에서 마이그레이션하는 작업은 `helm fetch` (헬름 2 CLI), `helm
chart save`, `helm chart push` 정도로 간단하다.
```

**Review by jmnote — COMMENTED** [review]
수고하셨습니다.

**Comment by markruler** [comment]
@bridgetkromhout 
I have signed-off on my commit. Why is there an DCO error?

**Comment by flynnduism** [comment]
> I have signed-off on my commit. Why is there an DCO error?

It's passing now

File: content/ko/docs/topics/registries.md [hunk 21]
```diff
@@ -0,0 +1,237 @@
...
+
+헬름 3는 패키지 배포를 위해 <a href="https://www.opencontainers.org/"
+target="_blank">OCI</a>를 지원한다. 차트 패키지는 OCI 기반 레지스트리 간에
+저장 및 공유할 수 있다.
+
+## OCI 지원 활성화
+
+현재 OCI 지원은 *실험적(experimental)* 으로 간주되고 있다.
+
+아래에서 기술된 명령들을 사용하려면 환경에서 `HOME_VERSION_OCI` 변수를 설정하자:
```
Comment (singlemocha) [inline content/ko/docs/topics/registries.md:16]: 아래에서 기술된 명령들을 사용하려면 환경에서 `HOME_EXPERIMENTAL_OCI` 변수를 설정하자.

Comment (markruler) [reply content/ko/docs/topics/registries.md:16]: 리뷰 감사드립니다. 오타 교정해서 반영하겠습니다.

**Comment by bridgetkromhout** [comment]
We seem to have another unsigned commit.

**Comment by bridgetkromhout** [comment]
Hi @markruler perhaps this link has the info needed? https://github.com/helm/helm-www/pull/825/checks?check_run_id=1132137156

**Comment by markruler** [comment]
@bridgetkromhout 
I keep trying in the same way as the info...

**Comment by bridgetkromhout** [comment]
> @bridgetkromhout
> I keep trying in the same way as the info...

Thanks. I see that you signed the commit, so I manually set it to pass. I think github may be having an intermittent failure on that check.

**Comment by markruler** [comment]
> Thanks. I see that you signed the commit, so I manually set it to pass. I think github may be having an intermittent failure on that check.

I don't know the exact reason, but is it probably merged with another branch? From now on, I will be careful of dependent documents so that I won't let the useless commit increase.. And I will reflect reviews through squash. Thanks!

File: content/ko/docs/topics/registries.md [hunk 22]
```diff
@@ -0,0 +1,236 @@
...
+
+헬름 3는 패키지 배포를 위해 <a href="https://www.opencontainers.org/"
+target="_blank">OCI</a>를 지원한다. 차트 패키지는 OCI 기반 레지스트리 간에
+저장 및 공유할 수 있다.
+
+## OCI 지원 활성화
+
+현재 OCI 지원은 *실험적(experimental)* 으로 간주되고 있다.
+
+아래에서 기술된 명령들을 사용하려면 환경에서 `HELM_EXPERIMENTAL_OCI` 변수를 설정하자:
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:15]:
```suggestion
아래에서 기술된 명령어들을 사용하려면 환경에서 `HELM_EXPERIMENTAL_OCI` 변수를 설정하자:
```

File: content/ko/docs/topics/registries.md [hunk 23]
```diff
@@ -0,0 +1,236 @@
...
+
+```console
+export HELM_EXPERIMENTAL_OCI=1
+```
+
+## 레지스트리 실행
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:21]:
```suggestion
## 레지스트리 구동
```

File: content/ko/docs/topics/registries.md [hunk 24]
```diff
@@ -0,0 +1,236 @@
...
+docker run -dp 5000:5000 --restart=always --name registry registry
+```
+
+이렇게 하면 `localhost:5000`에서 레지스트리 서버가 시작될 것이다.
+
+`docker logs -f registry`를 사용하여 로그를 확인하고 `docker rm -f registry`를
+사용하여 중지하자.
+
+스토리지를 보존하려면 위의 명령에 `-v $(pwd)/registry:/var/lib/registry`를
+추가하자.
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:36]:
```suggestion
스토리지를 보존하려면 위의 명령어에 `-v $(pwd)/registry:/var/lib/registry`를
추가하자.
```

File: content/ko/docs/topics/registries.md [hunk 25]
```diff
@@ -0,0 +1,236 @@
...
+
+자세한 설정 옵션은 [해당 문서](https://docs.docker.com/registry/deploying/)를
+참조하자.
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:39]:
```suggestion
자세한 설정 옵션은 [해당 문서](https://docs.docker.com/registry/deploying/)를
참고하자.
```

File: content/ko/docs/topics/registries.md [hunk 26]
```diff
@@ -0,0 +1,236 @@
...
+
+### 인증
+
+레지스트리에 인증을 활성화하려면, 다음과 같이 해보자.
+
+먼저 사용자 이름과 비밀번호로 `auth.htpasswd` 파일을 만들자.
```
Comment (jmnote) [inline content/ko/docs/topics/registries.md:45]:
```suggestion
먼저 사용자 이름과 비밀번호로 `auth.htpasswd` 파일을 만든다.
```

**Review by jmnote — COMMENTED** [review]
수고하셨습니다. 좀더 수정해봤습니다.

File: content/ko/docs/topics/registries.md (hunk 22 above)
Comment (markruler) [reply content/ko/docs/topics/registries.md:15]: 죄송하지만 다음 업데이트 PR을 만들어서 수정해도 될까요?
이 PR 커밋을 수정하면 DCO를 관리자들이 직접 수동으로 승인해줘야 하더라구요. 😂

Comment (jmnote) [reply content/ko/docs/topics/registries.md:15]: 그렇게 하시지요. 수고하셨습니다.

**Comment by jmnote** [comment]
lgtm

**Pull request merged**